src/eventPage/Event.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled from 'styled-components'
+
+export default function Event({event}) {
+const{eventname, eventlocation, eventdate} = event
```

Comment:
Better use camelcase for these destructured variables to increase readability for others
`eventName, eventLocation, eventDate`

Reply:
done :)